rs/registry/canister/src/mutations/node_management/do_update_node_ipv4_config_directly.rs

```diff
@@ -254,8 +215,8 @@ mod tests {
                 .expect("failed to get the node operator id");
 
         // create IPv4 config with invalid gateway IP
-        let mut ipv4_config = init_ipv4_config();
-        ipv4_config.gateway_ip_addr = "193.118.999.137".into();
+        let ipv4_config =
+            IPv4Config::try_new("193.118.59.140".into(), "193.118.999.137".into(), 29).unwrap();
```

Comment:
The problem of these tests is that they already fail here and not at line 226. Can you create an IPv4Config like this:
```
IPv4Config {
            "193.118.59.140".into(),
            "193.118.999.137".into(),
            29,
        }
```
And then try to call the `do_update_node_ipv4_config` and it should fail. Otherwise, the test just checks whether the `try_new` rejects invalid configs instead of making sure that the registry mutation actually rejects invalid configs.

Reply:
This is the same across all the tests that expect failure.